marketdata/stream/message.go

```diff
@@ -67,7 +75,15 @@ func (c *client) handleMessage(b []byte) error {
 	return nil
 }
 
-func (c *client) handleTrade(d *msgpack.Decoder, n int) error {
+type stocksMsgHandler struct {
```

Comment:
An interface compliance check would be nice for both `stockMsgHandler` and `cryptoMsgHandler`

Reply:
It's in `NewStocksClient()`.

Reply:
Then one could argue that `var _ StocksClient = (*stocksClient)(nil)` and `var _ CryptoClient = (*cryptoClient)(nil)` is not "necessary" either because of the `New...Client()` functions.
I'm fine with either solution but let's stick to one.

Reply:
👍  Added it for consistency.